### DISCUSSION TITLE
Where to report bugs/issues

### DISCUSSION THREAD

**Discussion by mhammond** [gulbanana/gg#4] [General]
This repo doesn't seem to have "issues" enabled, and none of these discussions topics seems to cover them. Where do we report problems?

(Specifically, on mac on 0.15.2, the "changed files" section can't be interacted with. A scroll bar is shown but it doesn't work (so I can't see the full list of changed files). Clicking on an individual changed file has no effect (so I can't see any file changes in a revision)

**Comment by gulbanana** [comment]
Feel free to open discussions about problems, or find me on Jujutsu's [discord](https://discord.gg/dkmfj3aGQN). At the moment I'm not sure whether I want to commit to having an official tracker - I'm working on GG but not necessarily at a predictable rate :)

File diffs, alas, are not displayed yet. I'll look into the broken scrollbar though, that's definitely a bug.

**Reply by mhammond** [reply]
Hey, no worries, I understand. You should think about asking for a channel on discord :)

I've an interesting problem with one specific repo too - gg fails with an internal error about watchman, which I do have installed, but this one repo in particular is failing! I'll maybe ask about that on discord as it seems like an internal jj issue more than gg. Are there logs available?

Cheers.

**Reply by gulbanana** [reply]
In the latest release, (which was just today, so you might need to update), running with —debug will produce some logging. I’m not sure whether if it’s tuned well enough to capture all issues, but it’s worth a shot.

**Comment by gulbanana** [comment]
just a final note before i close this (old-discussion hygiene): we do have a discord channel now, `#gg` on https://discord.gg/dkmfj3aGQN
